consensus/helpers.go

```diff
@@ -150,7 +166,15 @@ func (m *consensusModule) sendToNode(msg *typesCons.HotstuffMessage) {
 		m.nodeLogError(typesCons.ErrCreateConsensusMessage.Error(), err)
 		return
 	}
-	if err := m.GetBus().GetP2PModule().Send(cryptoPocket.AddressFromString(m.idToValAddrMap[*m.leaderId]), anyConsensusMessage); err != nil {
+
+	validators, err := m.getValidatorsAtHeight(m.CurrentHeight())
+	if err != nil {
+		m.nodeLogError(typesCons.ErrPersistenceGetAllValidators.Error(), err)
+	}
+
+	_, idToValAddrMap := typesCons.GetValAddrToIdMap(validators)
```

Comment:
I see your point but I could argue that it wouldn't be DRY, the logic inside the two resulting methods would be basically the same. We would just return a map or the other.

![image](https://user-images.githubusercontent.com/29378614/210964951-f06ab927-63f2-4327-a98e-5ffd5753571c.png)

If we had two methods like that you would have probably asked me to consolidate them :)

Maybe generics... but I feel like some reflection might be needed because `T ValAddrToIdMap | IdToValAddrMap` is not a core type (necessary to make the map).
It would probably look too complex for what it is simply a mapping....

I was about to reply with the above but I came up with this instead: https://github.com/pokt-network/pocket/pull/425/commits/fd6455dab9897bea61e62f00fd5cd5b435ce4838

![image](https://user-images.githubusercontent.com/29378614/210979329-a74921f1-e91c-41fd-83e2-a6d5f76d31a7.png)